scss/mixins/_forms.scss

```diff
@@ -95,20 +95,52 @@
         // Boosted mod: no box-shadow
 
         // Boosted mod: focus styles don't match validation state
+
+        // Boosted mod: apply focus mixin to allow sufficient contrast between error field focused and non focused
+        &:focus {
+          &[data-focus-visible-added] {
+            @include focus-visible($color: $border-color);
+          }
+        }
+        // End mod
       }
     }
 
     // Boosted mod: Remove border on input for form element QuantitySelector
-    .quantity-selector .form-control {
-      @include form-validation-state-selector($state) {
-        border-right: none;
-        border-left: none;
+    .quantity-selector {
+
+      .form-control {
+        @include form-validation-state-selector($state) {
+          // stylelint-disable-next-line scss/selector-no-redundant-nesting-selector
+          & ~ button {
+            border-color: $border-color;
+          }
+
+          &:focus {
+            &[data-focus-visible-added] {
+              ~ button,
+              ~ .btn:disabled {
+                border-color: var(--#{$prefix}btn-border-color) !important; // stylelint-disable-line declaration-no-important
+                border-width: $outline-width !important; // stylelint-disable-line declaration-no-important
+              }
+            }
+          }
+        }
+      }
 
-        // stylelint-disable-next-line scss/selector-no-redundant-nesting-selector
-        & ~ button {
-          border-color: $border-color;
+      button {
+        &:focus {
+          &[data-focus-visible-added] { // stylelint-disable-line selector-no-qualifying-type
+            &:first-of-type {
+              border-right: var(--#{$prefix}border-width) solid $border-color;
+            }
+            &:last-of-type {
+              border-left: var(--#{$prefix}border-width) solid $border-color;
+            }
```

Comment:
Find weird the right border here: 
![image](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/91960143/8c5a1c72-6816-4e89-94f0-9d1f65811c84)